training/model_evaluation/environments/spec.yaml

```
@@ -0,0 +1,5 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+name: AzureML_Model_Evaluation_Env
```

To be consistent with other environments, and because "AzureML" and "Env" are inherent:
```suggestion
name: model-evaluation
```